server/prisma/migrations/20240527134914_switch_to_uuids/migration.sql

```diff
@@ -0,0 +1,780 @@
+ALTER TABLE "BruteStartingStats" ADD CONSTRAINT "BruteStartingStats_pkey" PRIMARY KEY ("id");
+
+-- AlterTable
+ALTER TABLE "Clan" DROP CONSTRAINT IF exists "Clan_pkey";
+ALTER TABLE "Clan" ADD COLUMN "temp_id" INT;
+ALTER TABLE "Clan" ADD COLUMN "temp_masterId" INT;
+UPDATE "Clan"
+SET "temp_id" = "id",
+    "temp_masterId" = "masterId";
+ALTER TABLE "Clan" DROP COLUMN "id";
+ALTER TABLE "Clan" DROP COLUMN "masterId";
+ALTER TABLE "Clan" ADD COLUMN "id" uuid NOT NULL DEFAULT uuid_generate_v4();
+ALTER TABLE "Clan" ADD COLUMN "masterId" uuid  NOT NULL ;
+ALTER TABLE "Clan" ADD CONSTRAINT "Clan_pkey" PRIMARY KEY ("id");
+
+-- AlterTable
+ALTER TABLE "ClanPost" DROP CONSTRAINT IF exists "ClanPost_pkey";
+ALTER TABLE "ClanPost" ADD COLUMN "temp_id" INT;
+ALTER TABLE "ClanPost" ADD COLUMN "temp_threadId" INT;
+ALTER TABLE "ClanPost" ADD COLUMN "temp_authorId" INT;
+UPDATE "ClanPost"
+SET "temp_id" = "id",
+    "temp_threadId" = "threadId",
+    "temp_authorId" = "authorId";
+ALTER TABLE "ClanPost" DROP COLUMN "id";
+ALTER TABLE "ClanPost" DROP COLUMN "authorId";
+ALTER TABLE "ClanPost" DROP COLUMN "threadId";
+ALTER TABLE "ClanPost" ADD COLUMN "id" uuid NOT NULL DEFAULT uuid_generate_v4();
+ALTER TABLE "ClanPost" ADD COLUMN "threadId" uuid  NOT NULL ;
+ALTER TABLE "ClanPost" ADD COLUMN "authorId" uuid  NOT NULL ;
+ALTER TABLE "ClanPost" ADD CONSTRAINT "ClanPost_pkey" PRIMARY KEY ("id");
+
+-- AlterTable
+ALTER TABLE "ClanThread" DROP CONSTRAINT IF exists "ClanThread_pkey";
+ALTER TABLE "ClanThread" ADD COLUMN "temp_id" INT;
+ALTER TABLE "ClanThread" ADD COLUMN "temp_clanId" INT;
+ALTER TABLE "ClanThread" ADD COLUMN "temp_creatorId" INT;
+UPDATE "ClanThread"
+SET "temp_id" = "id",
+    "temp_clanId" = "clanId",
+    "temp_creatorId" = "creatorId";
+ALTER TABLE "ClanThread" DROP COLUMN "id";
+ALTER TABLE "ClanThread" DROP COLUMN "clanId";
+ALTER TABLE "ClanThread" DROP COLUMN "creatorId";
+ALTER TABLE "ClanThread" ADD COLUMN "id" uuid NOT NULL DEFAULT uuid_generate_v4();
+ALTER TABLE "ClanThread" ADD COLUMN "clanId" uuid  NOT NULL;
+ALTER TABLE "ClanThread" ADD COLUMN "creatorId" uuid  NOT NULL;
+ALTER TABLE "ClanThread" ADD CONSTRAINT "ClanThread_pkey" PRIMARY KEY ("id");
+
+-- AlterTable
+ALTER TABLE "DestinyChoice" DROP CONSTRAINT IF exists "DestinyChoice_pkey";
+ALTER TABLE "DestinyChoice" ADD COLUMN "temp_id" INT;
+ALTER TABLE "DestinyChoice" ADD COLUMN "temp_bruteId" INT;
+UPDATE "DestinyChoice"
+SET "temp_id" = "id",
+    "temp_bruteId" = "bruteId";
+ALTER TABLE "DestinyChoice" DROP COLUMN "id";
+ALTER TABLE "DestinyChoice" DROP COLUMN "bruteId";
+ALTER TABLE "DestinyChoice" ADD COLUMN "id" uuid NOT NULL DEFAULT uuid_generate_v4();;
+ALTER TABLE "DestinyChoice" ADD COLUMN "bruteId" uuid  NOT NULL;
+ALTER TABLE "DestinyChoice" ADD CONSTRAINT "DestinyChoice_pkey" PRIMARY KEY ("id");
+
+-- AlterTable
+ALTER TABLE "Fight" DROP CONSTRAINT IF exists "Fight_pkey";
+TRUNCATE TABLE "Fight";
```

Comment:
I'm not sure how this behaves when some other models depend on the rows you are truncating, you probably need to add a cascade here.

Try to test it after having generated a tournament and manually fought a few times, to make sure the migratio ndoesn't get rejected and everything is deleted correctly.